PDSA_System/Server/Controllers/BrukerController.cs

```diff
@@ -104,7 +104,7 @@ public async Task<ActionResult<List<Bruker>>> UpdateBruker(Bruker bruker)
         /*
          Deleter brukere etter AnsattNr
          */
-        [HttpDelete("/api/[controller]/admin/DeleteBruker")]
+        [HttpDelete("/api[controller]/admin/DeleteBruker/{AnsattNr}")]
```

Comment:
Mangler her også oof
```suggestion
        [HttpDelete("/api/[controller]/admin/DeleteBruker/{AnsattNr}")]
```